eth/gasprice/gasprice_test.go

```diff
@@ -136,7 +136,7 @@ func newTestBackend(t *testing.T, config *params.ChainConfig, numBlocks int, gen
 	engine := dummy.NewFaker()
 
 	// Generate testing blocks
-	_, blocks, _, err := core.GenerateChainWithGenesis(gspec, engine, numBlocks, 1, genBlocks)
+	_, blocks, _, err := core.GenerateChainWithGenesis(gspec, engine, numBlocks, config.FeeConfig.TargetBlockRate-1, genBlocks)
```

Comment:
just in case this matters, this changed from `0` to `config.FeeConfig.TargetBlockRate-1`=`1`

Reply:
I think this one is more aligned with the actual chain behaviour, so unless UTs are breaking I prefer these over 0.